### DISCUSSION TITLE
When sending text data

### DISCUSSION THREAD

**Discussion by xja** [localsend/localsend#593] [Discussion (English, International)] [answered]
I believe a multiline text box could do the job instead of a tick box to switch between two different text box forms.

How do you feel about it?

**Comment by Tienisto** [comment, marked as answer]
Yes, this sounds good. I will think about it

**Reply by xja** [reply]
I made a modification in this [commit](https://github.com/xja/localsend/commit/b017c7493ff05735867dabcd8b0469ad74cfc981) and tested it on my android phones. Is it correct to strip it off this way?
